lib/Resizable.jsx

```diff
@@ -91,16 +92,14 @@ export default class Resizable extends React.Component {
 
   render() {
     let p = this.props;
-    let className = p.className ?
```

Comment:
Could you please keep this?

I find it is useful to see in the DOM and to allow an easy extension point via CSS, rather than forcing consumers to tweak the `style` prop.